doc/changes/latest.inc

```diff
@@ -80,6 +80,8 @@ Bug
 
 - Fix bug with :func:`mne.preprocessing.ICA.find_bads_eog` when more than one EOG components are present by `Christian O'Reilly`_
 
+- Fix bug with :func:`mne.io.Raw.set_meas_date` to support setting `meas_date` to `None`. by `Luke Bloy`_
```

Comment:
```suggestion
- Fix bug with :func:`mne.io.Raw.set_meas_date` to support setting `meas_date` to `None`, by `Luke Bloy`_
```

Reply:
Both meas_date and None should be in double-backticks, not single backticks.